packages/fiori-app-sub-generator/src/types/state.ts

```diff
@@ -125,6 +125,10 @@ export interface State {
      * The template selected by the user, maybe FE or FF
      */
     floorplan: Floorplan;
+    /**
+     * ABAP CSN (Core Schema Notation) - for writing to .appGenInfo.json
+     */
+    abapCsn?: AbapCsn[];
```

Comment:
My assumption is this should not be part of the state - @IainSAP can you recommend a more appropriate home for this. 
For context, it is passed to app gen (from adaptors) and written directly to the `.appGenInfo.json` file

Reply:
What is .appGenInfo.json, where is it defined and what is it purpose? Is there a writer for it?

Reply:
On review, I think this is actually app gen info additional entries. I would propose that we define a new state property `appGenInfo` based on the existing read me type (we used to have a readMe property for this exact case where adaptors or extensions wanted to add additional information). Then adaptors can add entries (use `additionalEntries`) into the state via this `appGenInfo` property. This property `abapCsn` appears to be for informational purposes only, we dont use it to generate anything AFAIK and so should be added in the adaptor and then will get written via the `generateAppGenInfo` ( maybe renamed from `generateReadMe`)  without any code  in the generators.